backend/src/crd/controller/viewer/main.go

```diff
@@ -46,6 +46,8 @@ var (
 	namespace = flag.String("namespace", "kubeflow",
 		"Namespace within which CRD controller is running. Default is "+
 			"kubeflow.")
+	clientQPS   = flag.Float64("client_qps", 5, "The maximum QPS to the master from this client.")
+	clientBurst = flag.Int("client_burst", 10, "Maximum burst for throttle from this client.")
```

Comment:
For the default values, how did you decide on the values?

Reply:
This is the default value set in client-go, there will be no difference if user does't set it manually.

Reply:
Cool, thanks.